fastlane/lib/fastlane/helper/git_helper.rb

```diff
@@ -3,36 +3,38 @@ module Actions
     GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
 
     def self.git_log_between(pretty_format, from, to, merge_commit_filtering, date_format = nil, ancestry_path)
-      command = ['git log']
-      command << "--pretty=\"#{pretty_format}\""
-      command << "--date=\"#{date_format}\"" if date_format
+      command = %w(git log)
+      command << "--pretty=#{pretty_format}"
```

Comment:
Are we sure `pretty_format` and `date_format` don't have to be quoted? Always?

Reply:
Ah, https://github.com/fastlane/fastlane/blob/30be4096f3c9a84c9fee3cf4a73c71a7508313f4/fastlane/lib/fastlane/helper/sh_helper.rb#L121-L129 always felt a bit "magic" to me but now I think I get it: Array as param, String as param, many params (that then can be shelljoined).

Reply:
Right, I think it's trying to be compatible with [IO#popen](https://www.rubydoc.info/stdlib/core/IO.popen).